src/api-reference/user-provisioning/spend/spend-role-code-definition.md

```diff
@@ -77,14 +77,13 @@ The role code value that can be used for provisioning with the `Spend Role Exten
 |`SHD_PASSWORD_ADMIN`|Password Manager|The user assigned this role can update passwords for Expense users. <br>User will have read only access to the following fields on the User Details page in User Administration: Title, First Name, Middle Name, Nickname, Last Name, Suffix, and Email.<br>**Preventing Access**: A module property is available to restrict this role from changing password. Please contact SAP Concur directly to have the Password Access Restriction feature activated.<br>**NOTE:** The users with Expense and either Travel or Invoice have one password for all applications. When any of the Password Manager roles changes a password, it changes for all.|Y|Y|
 |`EXP_REIMBURSEMENT_AUDITOR`|Reimbursement Auditor|A user assigned this role can view Expense Pay functionality: Funding Accounts, Batch Configurations, Card Programs, Expense Pay Settings, Current and Historical Batch List, Daily Funding and Returned Amounts, Payment Demand List, Report Payees List, and Employee Banking Status|N|Y|
 |`EXP_REIMBURSEMENT_APPROVER`|Reimbursement Manager|A user assigned this role can fully manage (add, edit, delete) Expense Pay functionality: Funding Accounts, Batch Configurations, Card Programs, and Expense Pay Settings. <br>In addition, a user assigned this role can view the following Expense Pay details: Current and Historical Batch List, Daily Funding and Returned Amounts, Payment Demand List, and Report Payees List. <br>Only global Reimbursement Managers can create and view payment demands: Employee Banking Status|Y|Y|
 |`SHD_ROLE_ADMIN`|Role Administrator|A user assigned this role is granted access to the **Expense, Invoice,** and **Request** tabs through User Permissions.|Y|Y|
 |`N/A`|Role Builder|**This role is intended to be assigned to and used by SAP Concur internal staff only.**|N|Y|
-|`SHD_CONFIG_ADMIN`|Shared Configuration Administrator|**This role is intended to be assigned to and used by SAP Concur internal staff only.**|Y|Y|
+|`SHD_CONFIG_ADMIN`|Shared Configuration Administrator|**__This role is intended to be assigned to and used by SAP Concur internal staff only, with few exceptions.__**<br>The user assigned this role can **__fully manage__** (add, edit, delete):<br><ul><li>Feature hierarchies for Expense Reports and Vendor Invoices</li><li>Employee group configurations, including assignment of the employee form</li><li>Employee forms and fields, validations, and custom and connected lists</li><li>Expense and payment delegate configurations</li><li>Reimbursement currencies</li><li>Imaging settings</li><li>Accounting structure (ledgers)</li><li>Localization tasks</li><li>Configuration change log (view only)</li></ul>Hierarchy segment values are used to identify the hierarchy node for which this employee has approval rights. If blank, resolves to the global group.<br>**NOTE:** A user cannot be assigned the admin role and the restricted admin role.|Y|Y|
```

Comment:
When you say "A user cannot be assigned the admin role and the restricted admin role" should we explicitly say the `SHD_CONFIG_ADMIN_RO` so they know know what the restricted admin role is that is being mentioned?

Reply:
I can do "A user cannot be assigned the SHD_CONFIG_ADMIN and the SHD_CONFIG_ADMIN_RO".